template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/TwoFactorAuthentication.cshtml

```diff
@@ -5,44 +5,32 @@
 }
 
 <h4>@ViewData["Title"]</h4>
-@if (Model.HasAny2faProviders)
+@if (Model.HasAny2faProviders && Model.Is2faEnabled)
```

Comment:
Yes but people may want fallbacks. Or it's simply not suitable for their audience. Take the US social security web site, it's 2fa is SMS based, because really, you can't assume every user is going to have a smartphone.